locales/en/app.ftl

```diff
@@ -851,3 +863,26 @@ join-firefox = There is a way to protect your privacy. Join {-brand-name}.
 
 # Link title
 learn-more-link = Learn more.
+
+
+email-sent = Email Sent!
+
+# Form title
+want-to-add = Want to add another email?
+
+# This is part of a confirmation message that appears after a user has submitted
+# the form to add an additional email to Firefox Monitor.
+verify-the-link = Verify the link sent to { $userEmail } to add it to Firefox Monitor.
+
+# This is part of a confirmation message that appears after a user has submited the
+# form to add an additional email to Firefox Monitor. { $preferencesLink } is a link
+# to the Preferences page. The code and text for the link is generated elsewhere
+# using the {-preferences} string.
+manage-all-emails = Manage all email addresses in { $preferencesLink }.
+
+# This string is a header on the user preferences page and
+# appears above a check-box list of user options which allow
+# the user to choose whether or not they want to receive breach
+# alerts for all of their monitored email addresses to a single 
+# email address.
+breach-alert-notifications = Breach Alert Notifications
```

Comment:
Is this replacing "Global communications"? If so, why isn't the old string removed?

Reply:
It does replace "Global Communications" but I was under the impression we should not remove strings at this point.

Reply:
When are we going to remove the strings from the previous version, given we launched v2? That would be a good start ;-)

Reply:
The old string is unused, the moment you push this code out, we're either going to see English or the new translation. The only reason to keep it is that we only expose the strings, but not the code, so we can still push updates to production.